Framework/DataHandling/inc/MantidDataHandling/LoadMuonNexusV2Helper.h

```diff
@@ -22,21 +30,32 @@ NeXus::NXInt loadGoodFramesDataFromNexus(const NeXus::NXEntry &entry,
 std::vector<detid_t>
 loadDetectorGroupingFromNexus(const NeXus::NXEntry &entry,
                               const std::vector<detid_t> &loadedDetectors,
-                              bool isFileMultiPeriod);
+                              bool isFileMultiPeriod, const int periodNumber);
```

Comment:
Don't bother with a const here since were taking a copy, and the bool isn't const either
If the implementation wants to mess with it's own number let it